src/userscores/view/templatepaintview.cpp

```diff
@@ -63,16 +63,16 @@ QString TemplatePaintView::zoomOutSequence() const
 
 void TemplatePaintView::load()
 {
-    IMasterNotationPtr masterNotation = notationCreator()->newMasterNotation();
-    Ret ret = masterNotation->load(m_templatePath);
+    INotationProjectPtr notationProject = notationCreator()->newNotationProject();
+    Ret ret = notationProject->load(m_templatePath);
 
     if (!ret) {
         LOGE() << ret.toString();
     }
 
-    setNotation(masterNotation->notation());
+    setNotation(notationProject->masterNotation()->notation());
 
-    if (masterNotation) {
+    if (notationProject) {
```

Comment:
Something seems wrong here (though it seemed wrong already), the notation project gets deleted when the shared_ptr goes out of scope, but there are still references to it (via the setNotation call), which is causing MuseScore to crash doing almost anything now. @igorkorsukov